snmp/asterisk

```diff
@@ -13,6 +13,7 @@ then
     echo "<<<asterisk>>>"
     $ASCLI -rx "core show channels" | awk '/active calls/ { print "Calls=" $1 } /active channels/ { print "Channels=" $1}'
     $ASCLI -rx 'sip show peers' | awk '/sip peers/ { print "SipPeers=" $1 "\nSipMonOnline=" $5 "\nSipMonOffline=" $7 "\nSipUnMonOnline=" $10 "\nSipUnMonOffline=" $12}'
+    $ASCLI -rx 'iax2 show peers' | awk '/iax2 peers/ { gsub("\\[",""); gsub("\\]",""); print "Iax2Peers=" $1 "\nIax2Online=" $4 "\nIax2Offline=" $6 "\nIax2Unmonitored=" $6}'
```

Comment:
Is it expected to have 2 times $6 for 2 different variables ?

Reply:
Good catch.  No.  I'll fix it.